packages/demo/src/app/home/home.component.html

```diff
@@ -290,7 +290,7 @@ <h4 class="h5 bold">Automatic Migration ⚙️</h4>
                 appCopyToClipboard
                 class="d-block mt-1 p-3"
                 [languages]="['bash']"
-                highlight="npx ng update @swisspost/design-system-styles --from=5 --to=6 --migrate-only"
+                highlight="npx ng update @swisspost/design-system-migrations --from=5 --to=6 --migrate-only"
```

Comment:
Can you a line somewhere to install/update the package as well?

Reply:
This package should not be installed into projects, I wrote about that is the README: https://github.com/swisspost/design-system/pull/2270#discussion_r1392017815.

Reply:
Ah true, good!